CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Beer Garden Changelog
 
-# 3.26.3
+# 3.27.0
```

Comment:
Let's keep this 3.26.3 for the moment, so we can do a release without all of the other major changes.